cypress/support/fragments/data-export/dataExportResults.js

```diff
@@ -13,5 +13,24 @@ export default {
       }),
       MultiColumnListCell({ 'row': 0, 'column': defaultJobProfile }).exists(),
     ]);
-  }
+  },
+
+  verifySuccessExportResultCells() {
```

Comment:
can't see verification of 
"File name. File name should be in bold and **should be clickable**. **The name of the file should contain the name of the input file (with UUIDs) followed by job id**."

Reply:
I find string in result table by this condition. You can see it inside function:
`exportWithDefaultInstancesJobProfile`

Reply:
I can't see implementation for points:
-  should be clickable
- followed by job id
may be  in bold should be removed from TC in TestRail. Need to check with PO and Team

Reply:
About bold:
It is not possible to check this because I can't get css attribute "font-weight". This property inherited a lot of times and this element don't contain this property inside `document.querySelector().style.fontWeight`

About clickable:
We have a lot of tests in another teams with clicking on this filename. After clicking download .mrc file will start. It seems we have excess checks

Reply:
you can just check href availability